README.md

```diff
@@ -1,42 +1,58 @@
 # Bike-store
-
-<h2>1 stage.</h2>
-  Objetive:
-    Create the frontend for the client begins to publicize your business.
-    The main sitie will be index, about us, products, last items and  blog. <br><br>
-  Technologies used:
-    The web will be created with HTML5, CSS3 and JS. It will be responsive in this  project.<br><br>
-  
-<h2>2 stage.</h2>
- Objetive:
-    Develop the backend of the business.<br><br>
- Technologies used:
-    Ruby.<br><br>
+<h2>Version</h2>
     
-  <h3>The  requeriments are:</h3>
-    
-    1. Blog administration.
+<h2>Vision</h2>
+
+<h2>How is it used?</h2>
+
+<h2>Develop</h2>
+
+  <h3>stage 1.</h3>
+    Objetive:
+      Create the frontend for the client begins to publicize your business.
+      The main sitie will be index, about us, products, last items and blog. <br><br>
```

Comment:
Move products catalog as a new stage no . 2 .... so we can have the skeleton for later stages.